chapters/open_science.md

```diff
@@ -0,0 +1,516 @@
+
+## Checklist
+
+### Open data
+
+- [ ] Ensure your data is in a simple, standard format or formats which is machine and human readable.
+- [ ] Check, reformat or create metadata to clearly describe what the data is, how it was collected, and any associated strengths/weaknesses to someone that finds it.
+- [ ] Identify a relevant, easily discoverable repository or repositories to host your data, and upload it there
+- [ ] Assign your data a persistent identifier such as a DOI.
+
+### Open source software
+
+- [ ] Put your code in a freely accessible repository
+- [ ] Include a licence granting others the right to use, copy and modify your work. You can use [this](https://choosealicense.com/) website to help you pick the most appropriate licence for your project.
+- [ ] Include a readme file containing useful information about a project such as what it is, how to use/install it and how to run any tests.
+- [ ] If you want others to collaborate on the project include contribution guidelines.
+
+### Open hardware
+
+- [ ] Use open hardware where practical
+- [ ] Make detailed documentation and designs for any hardware you develop openly available
+- [ ] Include a licence granting others the right to use, copy and modify your work. You can use [this](https://choosealicense.com/) website to help you pick the most appropriate licence for your project.
+- [ ] Include a readme file containing useful information about a project such as what it is, materials used, etc.
+
+### Open access
+
+- [ ] Publish your research in an open access journal
+- [ ] Store a copy and/or preprint of your work in a freely accessible public repository
+
+### Open notebooks
+
+- [ ] Keep notes in an Electronic Lab Notebook
+- [ ] Make your notebooks publicly accessible online.
+
+## What to learn next
+
+If you haven't had a chance already, take a look at the chapter on version control, particularly the sections on GitHub in the latter half.
+
+## Further reading
+
+[This](http://book.openingscience.org.s3-website-eu-west-1.amazonaws.com/) book on open science has a great deal of interesting information on open science. For information specific to open source software [this](https://opensource.guide/) is a good place to look. For more information on DOIs and citing resources look [here](http://www.doi.org/index.html). If you want to take a look at an active open source project this textbook *is* one. You can be found on GitHub [here](https://github.com/alan-turing-institute/the-turing-way) and/or take a look at the project [website](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)
```

Comment:
Not sure we need to link to the Turing website here as it doesn't contain any of the book's content. Looks like the final sentence is unfinished; would something like "Head over to our [main page](https://github.com/alan-turing-institute/the-turing-way) to explore the rest of the project, and check out our [contributing guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) for details of how you can involved." be what you meant here?